### PR TITLE
Feature/landing page

### PR DESCRIPTION
**Pull request by arslanyousaf77**

### CODE REVIEW

File: app/assets/stylesheets/default_layout.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require bootstrap.css
+
```
Comment (abbasraza172) [inline app/assets/stylesheets/default_layout.css]: please remove extra line.

File: app/assets/stylesheets/landing_page_theme.css.scss [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+#sidebar .nav a.zoomTool{color:#fff} 
+.thumbnail>a{display:block; text-align:center}
+.carousel-caption {background: transparent;background: transparent;}
+#compairTbl td{text-align:center;}
+.justify{text-align:justify;}
+.table td[align=right] {text-align: right;}
+.thumbnail h5,.thumbnail p{text-align:center}
+.bs-docs-sidenav > li > a {border: 0; border-bottom:1px solid #e5e5e5;}
+.thumbnail{position:relative;}
+.tag {background:url(assets/img/new.png) no-repeat 0 0; position: absolute; display:block; top: -4px;right: -18px; height:48px; width:48px;}
```
Comment (abbasraza172) [inline app/assets/stylesheets/landing_page_theme.css.scss]: this is not correct. please update as we did on the start of file.

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -1,2 +1,7 @@
 class ApplicationController < ActionController::Base
+    before_action :set_subdomain
+
```
Comment (abbasraza172) [inline app/controllers/application_controller.rb:3]: please add code docs here as well.

File: app/views/shared/default/_lowerHeader.html.erb [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<div class="row">
+  <div class="span4">
+    <h1>
+      <a class="logo" href="index.html"><span>Twitter Bootstrap ecommerce template</span>
+        <%= image_tag "logo_coeus.png", alt: "logo", height: "70%", width: "70%"%> 
+        <%# <%= image_tag "logo-bootstrap-shoping-cart.png", alt: "logo"%>
```
Comment (abbasraza172) [inline app/views/shared/default/_lowerHeader.html.erb]: please remove commented code here.

File: app/assets/stylesheets/font-awesome1.css [hunk 5]
```diff
@@ -0,0 +1,1479 @@
+/*!
```
Comment (abbasraza172) [inline app/assets/stylesheets/font-awesome1.css]: maybe this can be deleted now no?

File: app/views/home/next.html.erb [hunk 6]
```diff
@@ -0,0 +1 @@
+<h3>Next</h3>
```
Comment (abbasraza172) [inline app/views/home/next.html.erb]: please add a new line at end.

File: app/views/shared/default/_carousel.html.erb [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+        <%= image_tag "landing-page/bootstrap-templates.png", alt:"bootstrap ecommerce templates"%>
+        <div class="carousel-caption">
+          <h4>Lockets</h4>
+        </div>
+      </div>
+    </div>
+    <a class="left carousel-control" href="#myCarousel" data-slide="prev">&lsaquo;</a>
+    <a class="right carousel-control" href="#myCarousel" data-slide="next">&rsaquo;</a>
+  </div>
+</div>
```
Comment (abbasraza172) [inline app/views/shared/default/_carousel.html.erb]: same as here . please add new line at end.

File: app/assets/stylesheets/font-awesome1.css (hunk 5 above)
Comment (arslanyousaf77) [reply app/assets/stylesheets/font-awesome1.css]: yes

**Review by abbasraza172 — APPROVED** [review]

**Pull request merged**